backend/package.json

```
@@ -6,7 +6,9 @@
   "repository": "git@github.com:KILTprotocol/web3-login-demo.git",
   "license": "BSD-4-Clause",
   "scripts": {
-    "start": "nodemon"
+    "start": "nodemon",
+    "start-prod": "node ./build/server.js",
+    "build": "npx tsc"
```

AFAIK npx is unnecessary for package.json scripts
```suggestion
    "build": "tsc"
```